### PR TITLE
Support Crystal 0.25.0

### PR DESCRIPTION
**Pull request by mamantoha**

### CODE REVIEW

File: src/i18n/backend/yaml.cr [hunk 1]
```diff
@@ -10,7 +10,7 @@ module I18n
       getter available_locales
       property translations
 
-      @translations = Hash(String, Hash(String, YAML::Type)).new
+      @translations = Hash(String, Hash(String, YAML::Any)).new
```
Comment (eliasjpr) [inline src/i18n/backend/yaml.cr]: I think `YAML::Any` should be replaced with `YAML::Any::Type`

**Comment by faustinoaq** [comment]
@TechMagister Please after merge this, release a new **minor** version `v0.2.0` (no patch `v0.1.1`), because crystal v0.25.0 has many breaking changes :sweat_smile:

File: src/i18n/backend/yaml.cr (hunk 1 above)
Comment (mamantoha) [reply src/i18n/backend/yaml.cr]: Well, no. `YAML.parse` returns an `YAML::Any`.

**Comment by TechMagister** [comment]
Done. Sorry for the delay.

**Comment by faustinoaq** [comment]
@TechMagister Thank you so much! :tada: 

/cc @robacarp

**Pull request merged**